Azure/sonic-buildimage
# [Wistron] Add 6512-32r platform support

#### Why I did it

1. Add 6512-32r support for Wistron platform
2. Update sw-to3200k for newer branch

#### How I did it

1. Add code in device and platform folder for 6512-32r
2. Update sw-to3200k code both in device and platform folder

#### How to verify it

Install on Wistron device and run command to verify

Signed-off-by: RogerChang <Roger_Chang@wistron.com>


#### Which release branch to backport (provide reason below if selected)



#### Description for the changelog
both verified in 202111 and 202012

#### Link to config_db schema for YANG module changes
N/A

#### A picture of a cute animal (not mandatory but encouraged)
